docs/usage/logo-brand-guidelines.md

```diff
@@ -0,0 +1,36 @@
+# Logo and brand guidelines
+
+This page explains how you may use the Renovate name, logo and branding.
+
+## Do not pretend to be the real Renovate app
+
+Avoid using our name, logo, or branding in a way that causes people to think you are the real Renovate app on a public platform.
+For example: do _not_ call your self-hosted version something like @realrenovatebot on GitHub.
+
+## Do not pretend to be a Renovate developer
+
+Avoid using our name, logo, or branding in a way that causes people to think you are a developer of Renovate.
+
+## Allowed uses of the Renovate name
+
+You are allowed to use the Renovate name:
+
+- to refer to the official Renovate app
+- as a nickname/shorthand, in contexts where it is clear you are referring to your self-hosted version
+
+## Allowed uses of the Renovate logo
+
+You are allowed to use our logo as:
+
+- an icon in your repository readme, that says you are using Renovate
+- part of a badge in your repository readme, that says you are using Renovate
+- an avatar image for your self-hosted version of Renovate, but give your bot a _different_ name
```

Comment:
A question regarding the following statement:

> [...] but vie your bot a _different_ name

When I use a [GitLab project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) as the bot account that runs Renovate, is it permitted to use the name `renovate[bot]` in conjunction with the Renovate logo? Its [username](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#bot-users-for-projects) would be something like `@project_123_bot_4ffca233d8298ea1` and its email address would be `project_123_bot_4ffca233d8298ea1@noreply.example.com`. For instance, GitLab's project landing page would render the following component

![image](https://github.com/renovatebot/renovate/assets/2206639/ad7f6d91-e800-4b59-b210-2f9a1e93ff3a)

where only the logo and name are be visible, but the link target of the name is the user page of the project access token

![image](https://github.com/renovatebot/renovate/assets/2206639/ee6c2526-4fe8-4393-8fdc-104803845107)

so it's clear that this is not an official Renovate bot user.

What's the policy in this case?

Reply:
It's ok to use it this way as it's a private group service / not shared

Reply:
@rarkins how about making an "official" unofficial avatar/logo that people can use for their version of Renovate bot? Then we can simplify the instructions:

- Only the "real" Renovate bot (Mend hosted app) may use the official logo
- If you are self-hosting Renovate, use this _different_ logo

The idea would be that the new unofficial logo is _related/on brand_ to the official logo, but still clearly different.